packages/components/Avatar/src/Avatar.types.ts

```diff
@@ -219,17 +230,34 @@ export interface AvatarTokens extends IBackgroundColorTokens, IForegroundColorTo
   /**
    * Avatar sizes:
    */
+  size16?: AvatarTokens;
   size20?: AvatarTokens;
   size24?: AvatarTokens;
+
+  /** Not applicable for @platform android */
```

Comment:
Maybe we should have tokens based on the platform? E.g. AvatarWin32Tokens and AvatarAndriodTokens. And AvatarTokens will have tokens based on the platform

Reply:
This pattern to have a superset type files is consistent across other code in repo, using documentation for indicating subset support matrix. I would prefer keeping it this way for similarity.

Reply:
But there are a lot of changes and other platforms might add even more changes so it'll become hard to read

Reply:
Taking this offline - this approach was the recommended/ agreed upon approach for handling type differences in endpoints.

Reply:
I am going to merge this PR changes. The suggestions on handling type differences are not specific to this code - and if needed there can be another change review to update this guidance across. There is more active work for this component pending, so I would account for needed change in those! Please feel free to add any additional post merge comments that I can track back.